.gitignore

```diff
@@ -10,3 +12,5 @@ projects/lambdaman/lambdaman
 metadata/cache/
 metadata/md5-cache/
 profiles/use.local.desc
+
+.gitignore
```

Comment:
This looks very wrong. `.gitignore` is a part of repository.

Reply:
I think I read somewhere many years ago that this was the recommended way to do things, but also maybe it was because I had a very specific thing I was trying to do.

Either way, I did not intend to modify this file in the repo at all, so I will revert the changes entirely.